example/javalib/basic/1-script/FooTest.java

```diff
@@ -1,5 +1,5 @@
 //| extends: [mill.script.JavaModule.Junit4]
-//| moduleDeps: [Foo.java]
+//| moduleDeps: [./Foo.java]
```

Comment:
Looks like we should be handle this automatically.

Reply:
That's an unusual pattern in the JVM ecosystem, IMHO, esp. in Scala, where even `import mill` means a potentially relative path, unless you write `import _root_.mill`.